src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/PublicationSummaryViewModel.cs

```diff
@@ -0,0 +1,14 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+
+public record PublicationSummaryViewModel
+{
+    public required Guid Id { get; init; }
+    
+    public required string Title { get; init; }
+    
+    public required string Slug { get; init; }
+    
+    public required string Summary { get; init; }
+    
+    public required DateTimeOffset LastPublished { get; init; }
```

Comment:
We need docblocks for each property to generate the API docs. Most of the text should be sourced from the existing docs.

I forgot to point this out for the existing publication and pagination view models as well, so we should go back and sort them out too!

Reply:
Added! :)